### ISSUE TITLE
[P2][Sourcing][Accessibility][Platform Multi combobox] existing selected values count not being read

### ISSUE THREAD

**Issue by I543348** [SAP/fundamental-ngx#13119]
Is this a bug, enhancement, or feature request?
Bug

Describe your proposal.
When user has already selected few tokens in platform multi combobox and disabled user tries to focus on this control then selected number of values should be ready by screenreader when focus is on input element

Can you handle that on the application side
No

Which versions of Angular and Fundamental Library for Angular are affected? Please, specify the exact version. (If this is a feature request, use current version.)
ng: 15
core/platform: 0.43.51

If this is a bug, please provide steps for reproducing it; the exact components you are using;
https://ng-15-downport--fundamental-ngx.netlify.app/#/platform/multi-combobox

Please provide relevant source code (if applicable).
https://ng-15-downport--fundamental-ngx.netlify.app/#/platform/multi-combobox

Please provide stackblitz example(s).

In case this is Accessibility related topic, did you consult with an accessibility expert? If not, please do so and share their recommendations.
reported by ATL team

Did you check the documentation and the API?
Did you search for similar issues?
Is there anything else we should know?
IMPORTANT: Please refrain from providing links or screenshots of SAP's internal information, as this project is open-source, and its contents are accessible to anyone.

**Comment by mikerodonnell89** [comment]
Should add this to core multi-combobox as well while we're at it
